### ISSUE TITLE
v0.21.0: "error: memory read failed for 0x7391004200"

### ISSUE THREAD

**Issue by asmt3** [opentok/opentok-react-native#610]
# Bug Report

**Current behavior**
> (Summarize the bug encountered concisely)
Since upgrading from 
"opentok-react-native": "^0.20.1" to "opentok-react-native": "^0.21.0",

When I try to start a session

I get a runtime error "memory read failed at 0x7...." and the app crashes while debugging via XCode. I haven't tried a release build yet.

Reverting to 0.20.1 makes the crash not happen

iOS only. The crash does not happen on Android. 

**Steps to reproduce**

Upgrade to  the library mentioned above. I am using CallKit via CallKeep. 

**What is the current *bug* behavior?**
Crash with error message as above

**What is the expected *correct* behavior?**
Video session goes ahead

**Relevant logs and/or screenshots**

This error message is not relevant

```
AudioSessionServerImp.mm:305   { "action":"cm_session_begin_interruption", "error":"translating CM session error", "session":{"ID":"0x2320b","PID":424,"name":"HelloSelfMobileM"}, "details":{"calling_line":3464,"error_code":-12983,"error_string":"Insufficient priority"} }
```

The same log error message appears even after downgrading to 0.20.1

**Comment by enricop89** [comment]
Thanks for filling the issue. Can you reproduce it with https://github.com/opentok/opentok-react-native-samples?

**Comment by enricop89** [comment]
Duplicate #613

**Comment by enricop89** [comment]
I published a new version `0.21.1` that fixes the iOS bug. Let me know!

**Comment by asmt3** [comment]
Thank you. Will test :)
